release/_bom.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
 
+function download_bnd_files {
+	if [ ! -e "${_BUNDLES_DIR}/osgi/modules/biz.aQute.remote.agent-6.4.0.jar" ]
+	then
+		lc_download "https://repo1.maven.org/maven2/biz/aQute/bnd/biz.aQute.remote.agent/6.4.0/biz.aQute.remote.agent-6.4.0.jar" "${_BUNDLES_DIR}/deploy/biz.aQute.remote.agent-6.4.0.jar"
+	fi
+
+	lc_download "https://repo1.maven.org/maven2/biz/aQute/bnd/biz.aQute.bnd/6.4.0/biz.aQute.bnd-6.4.0.jar" "${_BUILD_DIR}/boms/biz.aQute.bnd-6.4.0.jar"
```

Comment:
@pgydev, why are you checking the existence of `biz.aQute.remote.agent-6.4.0.jar` before download it and not doing the same to the file `biz.aQute.bnd-6.4.0.jar`? Can you please clarify?

Reply:
I see, but, according to your suggested code, the `catalina.sh` is just being invoked after you perform this download. In light of this, is this check still necessary? Please see the first lines of the function `generate_distro_jar`. :slightly_smiling_face: